mwm-5-dynamic-delivery/presentation/PITCHME.md

```diff
@@ -418,6 +418,26 @@ private val listener = SplitInstallStateUpdatedListener { state ->
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Access</span><span style="text-transform: none; font-size:0.8em;"> to resources</span>
+
+```
+android {
+	...
+    buildTypes {
+        release {
+        	shrinkResources true
```

Comment:
Here it's the base gradle or dynamic one? Should be false?

Reply:
I added the explanation: for both.